keras_nlp/models/distilbert/distilbert_models.py

```diff
@@ -62,7 +63,7 @@ class DistilBertCustom(keras.Model):
     Examples:
     ```python
     # Randomly initialized DistilBERT encoder
-    model = keras_nlp.models.DistilBertCustom(
+    model = keras_nlp.models.DistilBert(
```

Comment:
You might want the example to follow changes you requested in the [`Bert` docstring](https://github.com/keras-team/keras-nlp/blob/da490c962a70a7246372ceeeb084d28b123769c6/keras_nlp/models/bert/bert_models.py#L63) where the data is defined first.